docker-compose.yaml

```
@@ -1,35 +1,13 @@
     image : infront/coughsoundsound-project
     build : . 
     volumes:
-     - ./:/code 
-    environment:
-      - DBHOST=dhricst.database.windows.net
-      - DBPASSWORD=P@ssw0rd
-      - DBPORT=
-      - DBNAME=coughsounddb
-      - DBUSER=cstadmin
-
+     - ./:/code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)